lib/Minilla.pm

```diff
@@ -109,6 +109,10 @@ Checking git's untracked files. If there's untracked files, minilla will abort.
 
 This step will run the `git remote`. If there's no remote, minilla will abort.
 
+=head2 CheckReleaseBranch
+
+Cheking git's current branch is `release.branch` key in minil.toml. If no match, minilla will abort.
```

Comment:
```suggestion
Checking git's current branch is `release.branch` key in minil.toml. If no match, minilla will abort.
```

Reply:
Thanks for the review. Fixed it: https://github.com/tokuhirom/Minilla/pull/305/commits/aede4f7fcad8628704f84419c619cd50d9325962